frontend/src/components/pages/details/components/DetailsCard/DetailsCard.tsx

```diff
@@ -26,9 +27,14 @@ export interface DetailsCardProps {
   className?: string;
   redirectionUrl?: string;
   type?: string;
+  viewPoints?: ViewPoint[];
+}
+
+interface DetailsCardPropsWithHandleViewPointClick extends DetailsCardProps {
+  handleViewPointClick?: (id: string) => void;
 }
```

Comment:
Can't we just have `handleViewPointClick` as an optional prop? Is `DetailsCardProps` without `handleViewPointClick` still used?

Reply:
Totally right, fixed!